src/Food/food.py

```diff
@@ -1,8 +1,16 @@
+from datetime import date, timedelta
+
 class Food(object):
     def __init__(self, name, expiration_date, use_by_length):
         self.name = name
-        self.expiration_date = expiration_date
-        self.use_by_length = use_by_length
+        self.expiration_date = date.fromisoformat(expiration_date)
+        self.use_by_length = timedelta(days = use_by_length)
+        self.isOpen = False
     
-    def openItem():
-        pass 
\ No newline at end of file
+    '''Updates self.expiration_date to self.use_by_length according to date food item is opened. 
```

Comment:
I think this comment should be adjusted slightly. We're updating `self.expiration_date` to the current date plus `self.use_by_length` and updating self.isOpen. I think saying "If date is changed, self.isOpen updates to True." Isn't exactly accurate basically.

Reply:
Done!